app-config.yaml

```diff
@@ -437,3 +442,6 @@ azureDevOps:
   host: dev.azure.com
   token: my-token
   organization: my-company
+
+apacheAirflow:
```

Comment:
What's the reasoning behind having an entry both in the `proxy` section and it's own `baseUrl`? I guess it for linking out but just want to check.

Thinking that we should just really use the proxy and `discoveryApi` instead of duplicating it.

Reply:
That's right - the `baseUrl` is used in generating links to the Airflow UI. This was modeled after the _sonarqube_ plugin.

I'll look into using the `discoveryApi`!

Reply:
Hmm yeah, actually thinking about it you probably will need to have a link to the original endpoint unless you get it back from the API in anyway. You don't want to link to the proxy URL directly from somewhere in Backstage I guess.

Reply:
Sounds good!

In the original issue there was also a request to support multiple Airflow instances, and I think having the separate Airflow config will be needed for that down the road. Similar to how Jenkins plugin has an array of _instances_ defined for its config.

Reply:
Yeah I mean the Jenkins one is a good example, I think that it moved from a similar approach of using the proxy backend to writing its own backend plugin that could proxy the requests for you to the right place and be a place to house the multiple sources logic. It's a good example to follow I think